src/support.sch

```diff
@@ -145,6 +172,37 @@
         </rule>
     </pattern>
 
+    <pattern id="graphic">
+        <rule context="graphic[not(starts-with(@xlink:href, 'http://') or starts-with(@xlink:href, 'https://'))]" role="warn">
+            <assert test="true">
+                &lt;<name/>&gt; without an absolute HTTP @xlink:href is ignored.
+            </assert>
+        </rule>
```

Comment:
Presuming this currently only allows an HTTP URL because that's only what Libero can currently handle?

Most publishers' `@xlink:href` (here at the point of validation) would not be resolvable URL, but rather a file reference. 

In the DAR format for example, (what Texture will use), this attribute value will be populated by the filename of the figure file, since it's kept in the same folder. Other publishers may have formats in which figure files are kept in an images folder in the same zip, so the `@xlink:href` might be 'images/filename.jpg'.

Either Libero will have to parse the full URL and this validation would have to occur on a pre-transformed file, or after ingestion, or we will have to change this rule to account for filename type `@xlink:href` values.

Reply:
As part of https://github.com/libero/libero/issues/131 we'll be adding support for eLife/DAR-style packages, part of that will be to turn these into absolute URIs. Internally Libero will be using HTTP URIs as they'll need to be visible in a browser.